slack/ecs-deployment-failure/lambdas/main.rb

```diff
@@ -0,0 +1,98 @@
+require 'json'
+require 'net/http'
+
+FAILURE_EVENT_NAME = 'SERVICE_DEPLOYMENT_FAILED'.freeze
+
+def handler(event:, context:)
+  puts "[INFO] #{event}"
+  puts "[INFO] Context: #{context}"
+  event_name = event.fetch('detail').fetch('eventName')
+
+  return unless deployment_failure?(event_name)
```

Comment:
```suggestion
  return unless event_name == FAILED_EVENT
```

NIT: kinda pointless to have a 1 LOC method. Using a more expressive const name also solves the problem